modules/ROOT/pages/managing-apis.adoc

```diff
@@ -41,49 +40,54 @@ To add APIs to your portal:
 
 . From the *API management* page, select the *APIs in Exchange* tab.
 . To search for an API by name or description, enter the name or description into the *Search* bar.
-. Select an API from the list, click the more options icon, and select *Add to portal*.
+. Locate the API in the list, click *+ (Add to the portal)*.
 +
 The API displays in the list in the *APIs in the portal* tab.
 
 == Remove APIs from the Portal
 To remove APIs from your portal:
 
 . From the *API management* page, select the *APIs in the portal* tab.
-. To search for an API by name or description, enter the name or description into the *Search* bar.
-. Select an API from the list, click the more options icon, and select *Remove from portal*.
+. To search for an API, enter the name, description, category, or tag keyword into the *Search* box.
+. Locate the API in the list and click the trash can (Remove from the portal).
 . Click *Confirm*.
 +
 In the *APIs in the portal* tab, the API is removed from the list.
 
 == Make API Versions Visible
 By default, when you add a new API, the minor version shared with an Administrator is the latest version of the API. The API instance visibility is set to Public instances. Only the API instances that are public in Exchange are available for end-users to request access to them.
 
-You can manage which version of an API a user can view in your portal. You can set the visibility to Administrators, Members, Everyone, None.
+You can manage which version of an API a user can view in your portal. You can set the visibility to Administrators, Members and user groups, Guests, or None.
 
 * Administrators 
 +
 Use this option if you want the version to be seen by the portal administrator only.
 * Members 
 + 
-Use this option if you want the version to be seen only by logged-in members.
-* Everyone
+Use this option if you want the version to be seen only by logged-in members or specific user groups.
+* Guests
 +
 Use this option if you want the version to be publicly available to anyone.
 
 * None
 +
-Use this option if you don't want the version to be seen by anyone.
+Don't select any users if you don't want the version to be seen by anyone.
```

Comment:
Can you reword to be positive?
Select no users to hide the version from everyone?